Public/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
this should be on github, we should build our app using react build script

after that it should generate a dist file, which should NOT be on github, but that dist file will work in browser